packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -178,14 +181,6 @@ export const CollectionScreenDetailsTile = ({
   const firstTrack = useSelector(selectFirstTrack)
   const messages = getMessages(isAlbum ? 'album' : 'playlist')
   useRefetchLineupOnTrackAdd(collectionId)
-  const collection = useSelector((state) =>
```

Comment:
this results in collection === null because the collection is a search collection. Then we try to destructure a collection that doesn't exist, leading to the bug. Essentially i just moved all these computations to the root component.